src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -64,7 +65,7 @@ def key_fields(cls) -> Tuple[str, Optional[str]]:
 
     def get_vm(self) -> VM:
         vm = VM(
-            name="proxy-%s-%s" % (self.region, self.proxy_id),
+            name="proxy-%s" % base58.b58encode(self.proxy_id.bytes).decode(),
```

Comment:
Could you document the motivation for this, and the max length you observed?

Reply:
added validation with comments.